plasmapy/particles/_special_particles.py

```diff
@@ -28,14 +28,13 @@ class _ParticleZooClass:
 
     Examples
     --------
-    >>> ParticleZoo = _ParticleZooClass()
+    >>> ParticleZoo = ParticleZoo()
     >>> 'e-' in ParticleZoo.leptons
     True
     >>> 'nu_e' in ParticleZoo.antineutrinos
     False
     >>> 'mu+' in ParticleZoo.antiparticles
     True
```

Comment:
```suggestion
    >>> par_zoo = ParticleZoo()
    >>> 'e-' in par_zoo.leptons
    True
    >>> 'nu_e' in par_zoo.antineutrinos
    False
    >>> 'mu+' in par_zoo.antiparticles
    True
```

I only chose `par_zoo` so there wouldn't be any name shadowing with `particle_zoo` defined later in the file.